apps/resources/models.py

```diff
@@ -115,3 +116,29 @@ def get_most_tried(user=None, limit=1):
             '-most_tried',
         )[:limit]
         return resources
+
+    def get_related(self, user=None, limit=3):
```

Comment:
This method is a bit confusing, some simplification and Pythonicification might be useful.